data/users.js

```diff
@@ -676,3 +676,7 @@ const userData = [
     ]
   }
 ];
+
+if (typeof module !== 'undefined') {
```

Comment:
I'm curious if we actually need this, but it didn't seem to break anything so I guess it's fine.